model/model.go

```diff
@@ -14,11 +14,29 @@ type Event struct {
 
 type Team struct {
 	Id               string
-	EventTag         string
 	Email            string
 	Name             string
 	Password         string
 	CreatedAt        string
 	LastAccess       string
 	SolvedChallenges string
 }
+
+type Config struct {
+	Host 		string `yaml:"host,omitempty"`
+	AuthKey 	string `yaml:"auth-key,omitempty"`
+	SigninKey 	string `yaml:"signin-key,omitempty"`
+	DB struct {
+		Host string `yaml:"host,omitempty"`
+		User string `yaml:"user,omitempty"`
+		Pass string `yaml:"pass,omitempty"`
+		Name string `yaml:"db_name,omitempty"`
+		Port uint 	`yaml:"db_port,omitempty"`
+	} `yaml:"db,omitempty"`
```

Comment:
Check `omitempty` values for the variables, when you are placing `omitempty`, then all the variables which has `omitempty` should have default vaules set in source code.